### DISCUSSION TITLE
Macos Mssql when table edit column dosnt set Auto Generated

### DISCUSSION THREAD

**Discussion by birisi999** [dbeaver/dbeaver#13856] [Q&A]
#### System information: 
- Operating system (distribution) and version: macos 10.14.2
- DBeaver version : 5.3.3.201901202008
- Java version : embedded
- Additional extensions

#### Connection specification:
- Database name and version :  Mssql 14
- Driver name: mssql jdbc
- Do you use tunnels or proxies (SSH, SOCKS, etc)?

**Comment by crosoftzach** [comment]
I also have run into this issue on v5.3.2 running on OS X 10.14.2 against MSSQL 14. The Auto Generated boxes show as enabled but are not actionable. Cannot create a column with auto generated IDs.

**Comment by ttdjong** [comment]
Got the same problem.

![image](https://user-images.githubusercontent.com/3746553/56343998-ebf01000-61ee-11e9-9f9f-59dd5b64af6b.png)

**Comment by uslss** [comment]
existinng column cannot be altered to identity in MS SQL.
https://social.msdn.microsoft.com/Forums/sqlserver/en-US/04d69ee6-d4f5-4f8f-a115-d89f7bcbc032/how-to-alter-column-to-identity11?forum=transactsql
This option is available for newly created column only.
